test/inductor/test_torchinductor.py

```diff
@@ -1081,6 +1081,39 @@ def flip(x):
         actual = _run_and_assert_no_indirect_indexing(self, flip_opt, x)
         self.assertEqual(expect, actual)
 
+    def test__unsafe_masked_index(self):
+        def fn(a, mask, idx):
+            return aten._unsafe_masked_index(a, mask, idx, 1)
+
+        self.common(
+            fn,
+            (
+                torch.randn(8, device=self.device),
+                torch.tensor([True, False, True], device=self.device),
+                [torch.tensor([3, 9, -2], device=self.device)],
+            ),
+        )
+
+    def test__unsafe_masked_index_backward(self):
+        def fn(a, mask, idx):
+            return aten._unsafe_masked_index(a, mask, [idx], 1)
+
+        inp = [
+            torch.randn(8, device=self.device, requires_grad=True),
+            torch.tensor([True, False, True], device=self.device),
+            torch.tensor([3, 9, -2], device=self.device),
```

Comment:
These two don't make sense for the backward.

Reply:
I'm not sure I understand. Can you elaborate a bit?

Reply:
We can't take the backward of non-floating point values, and we already test these in the forward test.

Reply:
These are the indices which has to be integer values and `requires_grad` is only set for `self` which is a floating point value.

Reply:
The forward test previously runs a backward test too?